sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -50,11 +47,11 @@ public ConfigurationClient(string connectionString, ConfigurationClientOptions o
                 throw new ArgumentNullException(nameof(options));
 
             ParseConnectionString(connectionString, out _endpoint, out var credential, out var secret);
-
+            _apiVersion = options.Version;
             _syncTokenPolicy = new SyncTokenPolicy();
             _pipeline = CreatePipeline(options, new AuthenticationPolicy(credential, secret), _syncTokenPolicy);
 
-            _clientDiagnostics = new ClientDiagnostics(options);
+            ClientDiagnostics = new ClientDiagnostics(options);
```

Comment:
Pass `suppressNestedClientActivities` as true in `new ClientDiagnostics(options, true);` to disable nested scopes.